haystack/document_store/faiss.py

```diff
@@ -199,6 +199,14 @@ def update_embeddings(
         """
 
         index = index or self.index
+
+        if update_existing_embeddings is True:
+            if filters is None:
+                self.faiss_indexes[index].reset()
+                self.reset_vector_ids(index)
+            else:
+                raise Exception("update_existing_embeddings is not supported with filters.")
```

Comment:
Let's rephrase this to "update_existing_embeddings=True is not yet supported ..." to be crystal clear